src/braket/jobs/hybrid_job.py

```diff
@@ -203,6 +208,27 @@ def job_wrapper(*args, **kwargs):
     return _hybrid_job
 
 
+def _validate_python_version(aws_session: AwsSession, image_uri: str | None):
+    if image_uri:
+        print(
+            "Skipping python version validation, make sure versions match "
+            "between local environment and container."
+        )
+    else:
+        image_uri = retrieve_image(Framework.BASE, aws_session.region)
+        tag = aws_session.get_full_image_tag(image_uri)
+        major_version, minor_version = re.search(r"-py(\d)(\d+)-", tag).groups()
+        if not (
+            sys.version_info.major == int(major_version)
+            and sys.version_info.minor == int(minor_version)
+        ):
+            raise RuntimeError(
```

Comment:
Should this be an error or a warning?

Reply:
I believe it should be an error, as the behavior is very easily broken between python versions